ms-common-impl/src/main/java/net/trajano/ms/vertx/jaxrs/JwtSecurityContext.java

```diff
@@ -1,19 +1,17 @@
 package net.trajano.ms.vertx.jaxrs;
 
+import net.trajano.ms.core.JwtClaimsSetPrincipal;
+
+import javax.ws.rs.core.SecurityContext;
+import javax.ws.rs.core.UriInfo;
 import java.security.Principal;
 import java.text.ParseException;
 import java.util.Collections;
+import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.stream.Collectors'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)